onnx_tf/handlers/backend/scatter.py

```diff
@@ -0,0 +1,35 @@
+import tensorflow as tf
+
+from onnx_tf.handlers.backend_handler import BackendHandler
+from onnx_tf.handlers.handler import onnx_op
+from onnx_tf.handlers.handler import tf_func
+
+
+@onnx_op("Scatter")
+@tf_func(tf.scatter_nd)
+class Scatter(BackendHandler):
+
+  @classmethod
+  def version_9(cls, node, **kwargs):
+    tensor_dict = kwargs["tensor_dict"]
+    axis = node.attrs.get("axis", 0)
+    assert axis == 0, "Non-zero axis is not supported in Scatter."
```

Comment:
This will not pass the ONNX unit test, leaving our CI always in the failed state. Shall we come up with some general approach to take care the cases where we can only provide partial support to ONNX specs? For example, using strict mode to differentiate asserts vs warnings could be an option. Just do not like the fact that we will never pass the ONNX tests :-)

Reply:
In the past, we blacklist the particular onnx unit test we do not support.

Since the goal is to support strided slice, I don't see this restriction as a big issue at the moment. What do you think?

Reply:
We can put the op on the blacklist so it will not go through the ONNX unit test. That to me is not very elegant solution because we won't be testing the part that is supposed to work. The bottom line is I don't think we should merge a PR if we know it fails the ONNX unit test.

Reply:
If the goal of scatter is to support strided slice, not fully support the ONNX specs for scatter, I would add it to the blacklist as the short term solution.